### DISCUSSION TITLE
Welcome to vault-ai Discussions!

### DISCUSSION THREAD

**Discussion by pashpashpash** [pashpashpash/vault-ai#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gitardav** [comment]
Hello

Thank you for sharing your project with the community.

I am encouraged by this solution's ability to synthesize ChatGPT insights from personal documents and data. 

Cheers,

AD

**Comment by SaltedBone** [comment]
Hi. Yes, thanks for sharing & for starting a discussion! I'm actually here to learn something by hopefully integrating this with  Niek/chatgpt-web But I won't burden Y'all with that. Literally just ordered a laptop off Amazon it'll be here Wednesday. But that's me. Thanks.
